### PR TITLE
updating index page

### PR DESCRIPTION
**Pull request by Poolitzer**
Hey guys,

I tried to get into the project with this PR. I did some little typos fixes and updated the screenshot and promotion video.
If you guys feel like that looks alright or have feedback, I will have a look over the rest of the pages as well, getting everything up to date when needed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h3>the lightweight<br>YouTube experience<br>for Android</h3>
             <div id="text-1-wrapper">
                 <div class="row">
                     <div class="col-sm-6">
-                        <p><strong>Do you like watching videos on YouTube<br>but are looking for a simple app to do that?</strong></p>
+                        <p><strong>Do you like watching videos on YouTube<br>but look for a simpler app to do that?</strong></p>
```
Comment (TheAssassin) [inline index.html]: I think we should just put a `and` for that `but`, then the sentence makes more sense in general.
I'm not sure about changing present participle for simple present. What's the rationale behind this change?
I don't think we should put any sort of "weighting" into this sentence. Who says NewPipe is simpler than app XYZ? We need to be careful about this.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
                 <div class="header-text pull-left">
                     <h1 class="hidden-xs">NewPipe</h1>
                     <br class="hidden-xs">
-                    <h3>the lightweight<br>YouTube experience<br>for Android</h3>
+                    <h3>The lightweight<br>YouTube experience<br>for Android</h3>
```
Comment (TheAssassin) [inline index.html]: That change was long overdue. Thanks.

File: index.html [hunk 3]
```diff
@@ -177,7 +177,7 @@ <h3 style="font-weight: bold">NewPipe - the smart streaming solution</h3>
                                     </svg>
                                     <span class="circle-text"><i class="fa fa-signal"></i></span>
                                 </div>
-                                <p class="text-center">Little Data Consumption</p>
+                                <p class="text-center">Less Data Consumption</p>
```
Comment (TheAssassin) [inline index.html]: Less than...? Not sure about this change. @TobiGr what's your opinion?

File: index.html [hunk 4]
```diff
@@ -269,9 +269,9 @@ <h3>Subscriptions</h3>
                     <div class="feature-description">
                         <p><strong>Never miss updates again:</strong></p>
                         <ul>
-                            <li>Subscribe a channel with one click</li>
-                            <li>Use your custom feed</li>
-                            <li>Watch your favourite artists</li>
+                            <li>Subscribe to a channel with one click</li>
+                            <li>Use your customized feed</li>
```
Comment (TheAssassin) [inline index.html]: Custom feeds are feeds users created themselves. Customized feeds are feeds created by third parties with changes from the users.

The question is: are they *custom* or *customized*?

File: index.html [hunk 5]
```diff
@@ -269,9 +269,9 @@ <h3>Subscriptions</h3>
...
+                            <li>Watch your favourite channels</li>
```
Comment (TheAssassin) [inline index.html]: Makes sense in the YouTube context, but might not necessarily make sense for other services we support. Maybe we can find a more "media agnostic" term?

File: index.html [hunk 6]
```diff
@@ -484,7 +484,7 @@ <h3 class="title">Download NewPipe</h3>
                             <div class="col-xs-12 col-sm-6 col-lg-4" id="download-stable">
                                 <h4>F-Droid</h4>
                                 <div class="download-info">
-                                    <p class="download-version api-f-droid-stable-version">0.11.6</p>
+                                    <p class="download-version api-f-droid-stable-version">0.18.0</p>
```
Comment (TheAssassin) [inline index.html]: Those data are fetched dynamically via AJAX requests to our API. Hardcoding values in there doesn't make *any* sense really. We should put something in there like "You need to enable JavaScript in order to have up to date information". Otherwise, we need to fetch that information on build and re-render that page occassionally IMO.

CC @TobiGr, perhaps we need to open a new issue about this.

File: index.html [hunk 7]
```diff
@@ -540,8 +540,8 @@ <h4>F-Droid</h4>
 
                                             </div>
                                             <div class="item">
-                                                <p>F-Droid also guarantees the best possible privacy and therefore does not track its users. <br />
-                                                    At this point we would like to thank F-Droid for providing the downloads.
+                                                <p>F-Droid also guarantees the best possible privacy and does not track its users.<br />
```
Comment (TheAssassin) [inline index.html]: F-Droid don't guarantee anything, they try their best and do a great job IMO. Let's be a bit more realistic in this sentence and not make any promises.

File: index.html [hunk 8]
```diff
@@ -540,8 +540,8 @@ <h4>F-Droid</h4>
...
+                                                    At this point, we would like to thank F-Droid for providing their services.
```
Comment (TheAssassin) [inline index.html]: We should probably point out what services we use from them. Right now, it's pretty much building and distributing NewPipe APKs safely and reproducibly.

Also, you don't really "provide" downloads, you "host" them. But that's just my personal preference, I guess. We should reword the entire sentence.

File: index.html [hunk 9]
```diff
@@ -604,7 +604,7 @@ <h4>NewPipe is free and open source</h4>
                         </ul>
                         <p class="button-wrapper">
                             <a href="https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#newpipe-contribution-guidelines"><button class="action">Contribute</button></a>
-                            <a href="https://github.com/TeamNewPipe/NewPipe/"><button class="action"><i class="fa fa-star"></i> <span class="api-stargazers">3000</span></button></a>
+                            <a href="https://github.com/TeamNewPipe/NewPipe/"><button class="action"><i class="fa fa-star"></i> <span class="api-stargazers">7000</span></button></a>
```
Comment (TheAssassin) [inline index.html]: Another of these data we fetch live. But this value doesn't change so often, and is unlikely to go down suddenly. Perhaps append a little `+` to kind of show it's a potentially old value?

File: index.html (hunk 1 above)
Comment (Poolitzer) [reply index.html]: I can imagine the but comes from using their website/app.
Oh grr, now I have to argue in english grammar. I think it is a participle phrase I want here? The watching part describes the app more closely, which means we don't need the participle in the main phrase. Maybe replace look with want, that could help my point.

Tbh, lets rephrase that whole second sentence, I think simple is in general not appropriate.
Do you like watching videos on YouTube _but want an inutitive, feature-rich app for that?_

File: index.html (hunk 2 above)
Comment (Poolitzer) [reply index.html]: np ;P

File: index.html (hunk 3 above)
Comment (Poolitzer) [reply index.html]: less then normal app, since you can have sound only. Thats what I thought.

What about little otherwise? Litte in comparison to what? :D

File: index.html (hunk 4 above)
Comment (Poolitzer) [reply index.html]: Well, that is customized, since feed is the the whats new area, right?

I didnt realized I switched the entries around, but thats not a problem, right?

File: index.html (hunk 5 above)
Comment (Poolitzer) [reply index.html]: uploader? :D

Why exactly is that there anyway, under the header of subscriptions?

File: index.html (hunk 6 above)
Comment (Poolitzer) [reply index.html]: a new issue sounds good. I didnt spend time to hunt down that script, but it didnt work on my local website, showed this hardcoded values. Thats why I changed it.

File: index.html (hunk 7 above)
Comment (Poolitzer) [reply index.html]: I was thinking about it, but didnt dare to change that much on my own directly.

File: index.html (hunk 8 above)
Comment (Poolitzer) [reply index.html]: lets do it then.

Maybe we want to link to their terms in that sentence? https://f-droid.org/en/about/#terms-etc

_F-Droid respects your privacy. They don’t track you, or your device. We would like to thank them for providing their services in this way._

File: index.html (hunk 9 above)
Comment (Poolitzer) [reply index.html]: I will definitly add a little + for you 😘 
That fetching also didnt work on my local instance :(

**Comment by Poolitzer** [comment]
@TheAssassin you didnt say anything about the picture/video. Are they good?

File: index.html (hunk 6 above)
Comment (TobiGr) [reply index.html]: > it didnt work on my local website, showed this hardcoded values

That's because of CORS

> CC @TobiGr, perhaps we need to open a new issue about this.

Yes, there are some more things to consider before displaying a message like "You need to enable JavaScript in order to have up to date information". For example this long text cannot be displayed on the button because it would destroy the layout.

File: index.html (hunk 4 above)
Comment (TobiGr) [reply index.html]: The feed will receive a major update soon: https://github.com/TeamNewPipe/NewPipe/pull/2309
I'd suggest to alter this slide once that PR is merged.

Comment (Poolitzer) [reply index.html]: That looks great.

Whatever you want, not my call ;P next update sounds good

**Comment by TobiGr** [comment]
The video looks good. Your screencast is shorter but the size was doubled from 1.2MB to 2.6MB. Do you have the original uncompressed version? I can try to compress it the same way I compressed the old one.

File: index.html (hunk 9 above)
Comment (TheAssassin) [reply index.html]: Hmm... might be a cross-origin problem? We're not sending any CORS headers there. Mind to open a new issue?

File: index.html (hunk 6 above)
Comment (TheAssassin) [reply index.html]: One might just show nothing or a `-` or `?` or something.

File: index.html (hunk 5 above)
Comment (TheAssassin) [reply index.html]: You're not watching the uploader, you're watching their uploaded stuff. But you may also just listen, not watch, etc... That's kind of the issue.

> Why exactly is that there anyway, under the header of subscriptions? Not really related to that feature, I can just watch my favourite artists anywhere else in the app, right?

CC @TobiGr

File: index.html (hunk 1 above)
Comment (TheAssassin) [reply index.html]: We might add `and privacy friendly` to your suggestion, then it's perfect.

Comment (TobiGr) [reply index.html]: @TheAssassin That sounds good

File: index.html (hunk 3 above)
Comment (TheAssassin) [reply index.html]: Well, less in comparison to what? I think "little" has a less "comparative" attitude than "less".

Please consider we have a large user base which doesn't use any Google apps.

I know it's a bit tedious to discuss about such details. I would've suggested low or reduced, but all those are "comparative" as well...

File: index.html (hunk 5 above)
Comment (TheAssassin) [reply index.html]: `content feed` is a pretty technical but IMO the most precise term. But we address end users as well with the website. So that's not necessarily a good wording.

Let's keep it "channels".

File: index.html (hunk 3 above)
Comment (TobiGr) [reply index.html]: I'd rather keep little

File: index.html (hunk 9 above)
Comment (Poolitzer) [reply index.html]: I should be able to see this in the webconsole then right?

Comment (TheAssassin) [reply index.html]: See what? That there's some CORS issue? Depends on your browser, normally there should be some sort of error message somewhere.

Comment (Poolitzer) [reply index.html]: Yup, there is, see #129

**Comment by Poolitzer** [comment]
Sure. If it works out, tell me how, then I can do it on my own if needed.

[input.zip](https://github.com/TeamNewPipe/website/files/4014647/input.zip)

File: index.html (hunk 1 above)
Comment (Poolitzer) [reply index.html]: Roger that, will put this in my next commit.

I will mark this as resolved so its out of the overview, if thats okay. Otherwise scream and reopen it ;P

File: index.html (hunk 3 above)
Comment (Poolitzer) [reply index.html]: Okay. I don't like Little for a reason I cant put my finger on, but I can definitely see the issue with a comparison. Will revert that.

File: index.html (hunk 6 above)
Comment (Poolitzer) [reply index.html]: What about a question mark with the long JS required message as hover attribute?

File: index.html (hunk 8 above)
Comment (Poolitzer) [reply index.html]: Could I get feedback to that guys?

**Comment by TobiGr** [comment]
Thanks. I used [this config](https://user-images.githubusercontent.com/17365767/71733945-0cc6c700-2e4b-11ea-99b6-c54abe087a33.png) to compress the video.

**Comment by TheAssassin** [comment]
Next time please consider transcoding using VP9 and Opus, they're both free (as in speech) codecs.

I reached ~1.1M with two-pass encoding using the constant quality mode (could probably go even lower):

```sh
ffmpeg -y -i input.mp4 -vf scale=210:370 -c:v libvpx-vp9 -b:v 0 -crf 30 -pass 1 -an -f webm /dev/null
ffmpeg -y -i input.mp4 -vf scale=210:370 -c:v libvpx-vp9 -b:v 0 -crf 30 -pass 2 -c:a libopus output.webm
```

By the way, the scaling doesn't keep the aspect ratio there, at that width the height would normally be ~ 373px.

@Poolitzer can you please explicitly provide a license for that video? For instance, CC-0 would be very suitable. That way we can safely publish it.

**Comment by Poolitzer** [comment]
Uhm. Not sure how I can explicitly do that within this current context? Is it enough when I say that it is under that license? Do I have to re-upload it somewhere? Edit it in? :D

And can I get you guys to check out the comments to the file? There are like three conversations still open where you'd need to reply.

File: index.html (hunk 9 above)
Comment (Poolitzer) [reply index.html]: This is btw still open since we havent decided how to handle that case in the other comment, I personally like the + idea, so even people with JS disabled can see that content.

File: index.html (hunk 6 above)
Comment (TobiGr) [reply index.html]: I'd like to keep default values when possible. This works e.g. when displaying the GitHub stars (7100+) or supported languages (60+), but is completely non-sense when displaying  the current version. In this cases, we should try to link to sites where the information is also displayed like the GitHub releases page.

Please keep also in mind that it might be our fault that the api information is not available. In this case all users would see the "please enable javascript" message.

Hover titles are bad for users on touch devices. In this case we would need to display an icons to tell them that there is more info to be displayed when tapping on the text.

File: index.html (hunk 8 above)
Comment (TobiGr) [reply index.html]: > F-Droid respects your privacy. They don’t track you, or your device. We would like to thank them for providing their services in this way.

sounds good to me.

File: index.html (hunk 6 above)
Comment (Poolitzer) [reply index.html]: Can I just put an a tag inside the p, without breaking the replacement from the javascript? Then I'd have no problem linking that.

Comment (TobiGr) [reply index.html]: It is okay for now because we replace the `innerHTML` as you can see [here](https://github.com/TeamNewPipe/website/blob/fc735a839feef1527d746f9b801074dc33554a3f/js/web-api.js#L34). I have aslo implemented the option to add api-data as attribute, but this is not used atm.

Comment (Poolitzer) [reply index.html]: okay, I will do this then

File: index.html (hunk 8 above)
Comment (Poolitzer) [reply index.html]: done

File: index.html [hunk 10]
```diff
@@ -484,7 +484,8 @@ <h3 class="title">Download NewPipe</h3>
                             <div class="col-xs-12 col-sm-6 col-lg-4" id="download-stable">
                                 <h4>F-Droid</h4>
                                 <div class="download-info">
-                                    <p class="download-version api-f-droid-stable-version">0.11.6</p>
+									<!--This is the closest we get to the actual release on FDroids website-->
+                                    <p class="download-version api-f-droid-stable-version"><a href="https://f-droid.org/packages/org.schabi.newpipe/#screenshots">Latest version on F-Droid</a></p>
```
Comment (TobiGr) [inline index.html]: Created https://gitlab.com/fdroid/fdroid-website/merge_requests/481 to solve this :)

Comment (TobiGr) [reply index.html]: Correct URL is now: https://f-droid.org/packages/org.schabi.newpipe/#latest

Comment (Poolitzer) [reply index.html]: Unbelievable. This mad lad just did it. Will update accordingly.

Comment (Poolitzer) [reply index.html]: done

**Comment by B0pol** [comment]
IMO we should change this screenshot. NewPipe now supports subtitles download, and this screenshot doesn't show that.

**Comment by TheAssassin** [comment]
Please feel free to make a new one and share it here. Make sure there's no private data in that screenshot. It has to be generic and representative, so I'd recommend to show some random content from the latest trending channels or so if you need some example video to showcase features.

**Comment by B0pol** [comment]
Should I also screenshot it with comments or next videos ?
newpipe_download
I took it with next videos but realized when the first screenshot was made, there was no comment support.

**Comment by Poolitzer** [comment]
I went for the one with the videos in the background, and also changed the ticks to mirror the screenshot

**Comment by TheAssassin** [comment]
All my comments can be considered resolve. Please merge at your own discretion.

**Comment by TobiGr** [comment]
I just realized that your screenshots have a different aspect ratio. Can you please use a different display size in the emulator? Otherwise the bottom of the box jumps when clicking on Downloads.

features

**Comment by B0pol** [comment]
Is it with all screenshots?
[Here is the screenshot I took in 1920:1080 (= 960x2 : 540x2)](https://user-images.githubusercontent.com/58657617/72725276-3555fd00-3b7d-11ea-92d9-661651c51749.png)

**Comment by TobiGr** [comment]
Only this one screenshot is affected

**Comment by Poolitzer** [comment]
that works

**Comment by Poolitzer** [comment]
updated screenshot, merged changes from this master. Good to go?

**Comment by TobiGr** [comment]
Thank you

**Comment by TheAssassin** [comment]
Deployed.

**Comment by B0pol** [comment]
In downloads the image shown is adele - skyfall, and not selena gomez - rare, the screenshot with rare has caption download

**Comment by TobiGr** [comment]
That's how downloads looks like now:
image

**Comment by B0pol** [comment]
I see this on mobile

**Comment by TobiGr** [comment]
I guess your browser cached the website.

**Comment by Poolitzer** [comment]
ShouldIblameCaching.com
